stable/redis-ha/templates/redis-ha-service.yaml

```diff
@@ -5,12 +5,14 @@ metadata:
   labels:
 {{ include "labels.standard" . | indent 4 }}
   annotations:
-  {{- if .Values.serviceAnnotations }}
-{{ toYaml .Values.serviceAnnotations | indent 4 }}
+  {{- if .Values.service.annotations }}
+{{ toYaml .Values.service.annotations | indent 4 }}
   {{- end }}
 spec:
-  type: ClusterIP
+  type: {{ default "ClusterIP" .Values.service.type }}
+  {{- if (not .Values.service.type) }}
   clusterIP: None
```

Comment:
@DandyDeveloper It's likely because of these lines. The original default was: 
```
type: ClusterIP
clusterIP: None
```
But the new templating seems to template:
```
type: ClusterIP
```
This allows for a headless service by default. I think in order to get the same behavior you will need to comment out the .Values.service.type value.

Reply:
Good catch, quick fix. I'm just checking something with the LB Service as well. Won't take me long.

Reply:
@ssalaues Oh boy, don't ever forget to sign the commits... 

Fixed it I think. Pending CI passing.

Reply:
Still failing.... I'm going to test this thoroughly locally.

Reply:
@ssalaues Working. /cheer

Reply:
Awesome thanks!